gulp.config.js

```diff
@@ -0,0 +1,27 @@
+/* global module, __dirname */
+var APPLICATION_DIRECTORY = 'app/src',
+  CONNECTION_PORT = 8888;
+
+var config = {
```

Comment:
l initially had the configuration in the same file but I thought it looked a messy. I do agree it makes it less maintainable which I think is more important so I will move it back into the same file.